### DISCUSSION TITLE
Why does displacement remain

### DISCUSSION THREAD

**Discussion by FURUNZ** [SeisSol/SeisSol#1211] [General]
I don't understand why the displacement and stress of the surface will not disappear when simulating seismic wave propagation. My simulation time is set at 20s. When the numerical simulation ends, there is still displacement on the surface
![image](https://github.com/user-attachments/assets/03abb84e-afca-4e2e-865a-2cff212ff7ea)

**Comment by Thomas-Ulrich** [comment]
without more information, hard to answer. Open a full documented issue if you think there is a bug.

**Comment by FURUNZ** [comment]
I just simulated the displacement generated by a point source earthquake. I don't understand why the surface displacement never returned to 0 during my simulation. I tried tpv29 in the example, and the surface displacement in his simulation result did not turn to 0 at the end of the simulation. I wonder if this is normal？




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "SeisSol/SeisSol"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年11月19日(星期二) 下午3:29
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [SeisSol/SeisSol] Why does displacement remain (Discussion #1211)





 
without more information, hard to answer. Open a full documented issue if you think there is a bug.

**Reply by Thomas-Ulrich** [reply]
well if you impose a point source, it is expected that you get some remaining displacement in the near-field, no? See e.g. Shearer, p252

![image](https://github.com/user-attachments/assets/8fe35e08-f72e-4964-8837-646cc7a6feac)

**Comment by FURUNZ** [comment]
Ok, thank you. Is this related to the area size of the set point source, and is this a normal result of the source radiating near field stress waves?Whether it can be eliminated




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "SeisSol/SeisSol"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年11月19日(星期二) 下午3:40
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [SeisSol/SeisSol] Why does displacement remain (Discussion #1211)





 
well if you impose a point source, it is expected that you get some remaining displacement in the near-field, no? See e.g. Shearer, p252
 
image.png (view on web)
